example/HtmlOutput/html-example.php

```diff
@@ -8,7 +8,7 @@
 require __DIR__ . '/../example.php';
 
 // 2. Create a full payment part in HTML
-$output = new QrBill\PaymentPart\Output\HtmlOutput\HtmlOutput($qrBill, 'en');
+$output = new QrBill\PaymentPart\Output\MarkupOutput\HtmlOutput($qrBill, 'en');
```

Comment:
I am not a fan of introducing this BC break just to add a new output format, which brings no other change or benefit for people who are already using `HtmlOutput`.

How about we keep `HtmlOutput\HtmlOutput` and create a new `XslFoOutput\XslFoOutput`, yet both inherit from `MarkupOutput\AbstractMarkupOutput`?

For a v3 I'd see it more useful to remove the double information completely and just have `QrBill\PaymentPart\Output\HtmlOutput`

Reply:
Hi and thanks for reviewing. I agree, not a fan of the BC break either. This proposal was the most logical way to the inheritance but it's true that the new outputter probably does not justify the introduction of a BC break. 
Your proposed way of adding `XslFoOutput\XslFoOutput` and make it (together with `HtmlOutput\HtmlOutput`)  inherit from MarkupOutput\AbstractMarkupOutput does not seem too pretty but I agree that its probably for the best as opposed to the BC break.

I'll try to change that behaviour.

Reply:
The bc break was removed in https://github.com/sprain/php-swiss-qr-bill/pull/78/commits/2fb45864e516bae07221d4c59423a19c6365e188.

I dropped the MarkupOutput subfolder entirely since I felt it wasn't necessary for a single class. `AbstractMarkupOutput` now resides directly in `Sprain\SwissQrBill\PaymentPart\Output`.

Reply:
Thanks! I'll get back to this as soon as I get the time to do so.

Reply:
Great, thanks!